{"name": "test_TC002_invalid_login", "status": "broken", "statusDetails": {"message": "TypeError: wait_for_element_to_presence() takes 2 positional arguments but 3 were given", "trace": "self = <test_scritps.test_invalid_login.TestInValidLogin object at 0x08DAF690>\nrequest = <FixtureRequest for <Function 'test_TC002_invalid_login'>>\n\n    def test_TC002_invalid_login(self, request):\n        username = excel.get_data(Constants.EXCEL_PATH, \"Login\", 1, \"UserName\")\n        password = excel.get_data(Constants.EXCEL_PATH, \"Login\", 1, \"Password\")\n        title = excel.get_data(Constants.EXCEL_PATH, \"Login\", 1, \"Title\")\n        expected_error_message = excel.get_data(Constants.EXCEL_PATH, \"Login\", 1, \"ErrorMessage\")\n    \n        login = pm.get_login_page(request.node.driver)\n        login.login(username, password)\n        login.verify_title(title)\n>       actual_error_message = login.get_error_message()\n\ntest_scritps\\test_invalid_login.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <source.pages.loginpage.LoginPage object at 0x02E94A30>\n\n    def get_error_message(self):\n>       self.wait_for_element_to_presence(*self.__error_message)\nE       TypeError: wait_for_element_to_presence() takes 2 positional arguments but 3 were given\n\nsource\\pages\\loginpage.py:20: TypeError"}, "steps": [{"name": "login", "status": "passed", "parameters": [{"name": "userName", "value": "'admin'"}, {"name": "passWord", "value": "'admin'"}], "start": 1544857587176, "stop": 1544857587743}, {"name": "verify_title", "status": "passed", "parameters": [{"name": "expected_title", "value": "'actiTIME - Login'"}], "start": 1544857587743, "stop": 1544857587767}, {"name": "wait_for_element_to_presence", "status": "broken", "statusDetails": {"message": "TypeError: wait_for_element_to_presence() takes 2 positional arguments but 3 were given\n", "trace": "  File \"C:\\Users\\PriyaPramod\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 141, in impl\n    return func(*a, **kw)\n"}, "parameters": [{"name": "element", "value": "'xpath'"}], "start": 1544857587768, "stop": 1544857587768}], "start": 1544857587075, "stop": 1544857587807, "uuid": "0f34b33f-6de7-437d-a546-7059e0e1b6e7", "historyId": "ad5f56afce8506a8ec613e72b4d9b2ee", "fullName": "test_scritps.test_invalid_login.TestInValidLogin#test_TC002_invalid_login", "labels": [{"name": "host", "value": "TY-Pramod-Personal"}, {"name": "thread", "value": "7492-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_scritps.test_invalid_login"}]}